server/Makefile

```diff
@@ -375,7 +375,7 @@ plugin-mocks: ## Creates mock files for plugins.
 
 einterfaces-mocks: ## Creates mock files for einterfaces.
 	$(GO) install github.com/vektra/mockery/v2/...@v2.23.2
-	$(GOBIN)/mockery --dir channels/einterfaces --all --output channels/einterfaces/mocks --note 'Regenerate this file using `make einterfaces-mocks`.'
+	$(GOBIN)/mockery --dir einterfaces --all --output einterfaces/mocks --note 'Regenerate this file using `make einterfaces-mocks`.'
```

Comment:
Seems unrelated. Should we split to a different PR?

Reply:
I needed this to run `make einterfaces-mocks` and recreate the mock for `ObserveAPIEndpointDuration`.